ADMVEDU95/ADMVEDU95/Services/DI/Assembly/ViewBuilderAssembly.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ViewBuilderAssembly.swift
+//  ADMVEDU95
+//
+//  Created by Satishur, Oleg on 26.05.2021.
+//
+// swiftlint:disable function_body_length closure_parameter_position
+import Foundation
+import Swinject
+
+class ViewBuilderAssembly: Assembly {
+    func assemble(container: Container) {
+
+        let firebaseService = container.resolve(FirebaseServiceProtocol.self)!
```

Comment:
please no force unwrapping even in Assembly
it relates to every force unwrapping in Assembly